packages/components/Molecules/DataTable/DataTable.vue

```diff
@@ -104,6 +106,26 @@ export default {
     if (this.isRowOriented !== true) {
       this.responsiveData(this.items)
     }
+
+    /* When viewed on mobile
+     * If the first row of table is a the row header
+     * And the first column of the table is not a header
+     * And the table is row oriented
+     * Then the right column of the stackable columns cannot be a header element
+     */
+    this.useRowHeaderClass = (this.isRowOriented && this.isFirstRowHeader && !this.isFirstColHeader)
```

Comment:
This works fine but having long ternary operators like this doesnt make it very readable - especially when you are just returning a boolean. 

My preference here would be a [computed property](https://vuejs.org/v2/guide/computed.html), your approach in the mounted hook does not update - If the data is updated dynamically without remounting it will remain unchanged in the table (for example if we use this in the future with a dynamic data source).

Psuedocode
```javascript
computed: {
  useRowHeaderClass () {
    if (%% YOUR BOOLEAN LOGIC FOR TRUE STATE) {
      return true
    }
  }
  useColHeaderClass () {
    if (%% YOUR BOOLEAN LOGIC FOR TRUE STATE) {
      return true
    }
  }
}
```
You dont need to return false state as it will evaluate to undefined which is falsey - You can use this in your template exactly the same.

Reply:
Hey @dylankelly,

Changes made:

1) Logic moved to the computed property.
2) To remove some of the complex logic in the previous commits I've based it on the logic below.

**============================================================
NOTE Logic added only IF its required to be Bolded
============================================================
Scenario 1

SINGLE COLUMN
NOT BOLDED

Column oriented (group content by column) &&
Use first column as table header: FALSE &&
Use first row as table header: FALSE &&
============================================================
Scenario 2

DOUBLE COLUMN
NOT BOLDED

Column oriented (group content by column) &&
Use first column as table header: TRUE &&
Use first row as table header: FALSE &&
============================================================
Scenario 3 => Add this condition

SINGLE COLUMN
BOLDED

Column oriented (group content by column) &&
Use first column as table header: FALSE &&
Use first row as table header: TRUE
============================================================
Scenario 4 => Add this condtion

DOUBLE COLUMN
BOLDED

Column oriented (group content by column) &&
Use first column as table header: TRUE &&
Use first row as table header: TRUE
============================================================
Scenario 5

SINGLE COLUMN
NOT BOLDED

Row oriented (group content by row) &&
Use first column as table header: FALSE &&
Use first row as table header: FALSE
============================================================
Scenario 6 => Add this condition

SINGLE COLUMN
BOLDED

Row oriented (group content by row) &&
Use first column as table header: TRUE &&
Use first row as table header: FALSE
============================================================
Scenario 7

DOUBLE COLUMN
NOT BOLDED

Row oriented (group content by row) &&
Use first column as table header: FALSE &&
Use first row as table header: TRUE
============================================================
Scenario 8 => Add this condition

DOUBLE COLUMN
BOLDED

Row oriented (group content by row) &&
Use first column as table header: TRUE &&
Use first row as table header: TRUE
============================================================**